chart/values.yaml

```diff
@@ -321,6 +323,23 @@ backfill:
   tolerations: []
 
 
+deleteIndexes:
+  enabled: false
```

Comment:
you can already set it to true in prod and staging (maybe here too)

Reply:
Done in https://github.com/huggingface/datasets-server/pull/1536/commits/f38ba31e3b7664ee3322416f71f07444cfddf906